invisible_cities/cities/beersheba_test.py

```diff
@@ -103,7 +108,9 @@ def test_beersheba_exact_result( deco
                 assert_tables_equality(got, expected, rtol=1e-6)
 
 
-@mark.filterwarnings("ignore:.*not of kdst type.*:UserWarning")
+@ignore_warning.no_config_group
+@ignore_warning.str_length
+@ignore_warning.not_kdst
```

Comment:
It feels a bit odd to have separate `ignore_warning`s setup on each test function. 

Would it be possible to setup some sort of child class of the `ignore_warning` class that includes all the standard warnings we want to avoid required  (`no_config group`, `str_length`, `not_kdst`, and possibly `no_hits`) and apply it as a general filter to most of these functions? 

This is assuming that applying an `ignore_warning.not_kdst` to a function that doesn't currently have it wont be an issue, which I could be wrong about.

Reply:
That could hide warnings that might be meaningful to the developer. There is already a way to silence warnings on a global level in the `pytest.ini` file, but that would silence them everywhere, even where we don't expect one. This way we document which warnings are expected while showing that we understand them (hopefully).